src/SDL2.cs

```diff
@@ -8747,5 +8867,154 @@ public static unsafe int SDL_OpenURL(string url)
 		}
 
 		#endregion
+
+		/// <summary>
+		/// predefined SDL colors identical to those from System.Drawing.Color 
+		/// </summary>
+		public static class SDL_Colors
```

Comment:
Definitely outside the C spec, probably useful as a standalone class for moving away from System.Drawing